src/main/scala/is/hail/driver/Count.scala

```diff
@@ -31,8 +29,8 @@ object Count extends Command {
     val vds = state.vds
 
     val (nVariants, nCalledOption) = if (options.genotypes) {
-      val (nVar, nCalled) = vds.rdd.map { case (v, (va, gs)) =>
-        (1L, gs.count(_.isCalled).toLong)
+      val (nVar, nCalled) = vds.rdd.map { case (v, (va, gs: GenotypeStream)) =>
```

Comment:
The `Iterable[Genotype]` may not be a `GenotypeStream`, for example, if we ran `filter_genotypes.`  You should make a helper function somewhere that takes an `Iterable[Genotype]` and returns a `Iterable[Int]` and uses `hardcallIterator` if it is a `GenotypeStream` and `gs.iterator.map(_.unboxedGT)` otherwise (we'll, you'll have to create an IntIterator).